trace_chewer

```diff
@@ -71,6 +87,14 @@ def simplify():
                 while j < len(t) and t[j].startswith("  "): j += 1
                 del t[i:j+1]
             elif t[i].startswith("guardsuccess"): del t[i]
+            elif t[i].startswith("  ; "):
```

Comment:
Maybe a comment here stating that this parses the debug comments in the trace, couldn't hurt?

Reply:
I thought it was kinda obvious since ";" is the comment marker in LLVM IR?

Reply:
If you're super familiar with LLVM traces. I wouldn't say it's suuuper obvious and could be more explicit but it's not a hill I'm going to die on.

Reply:
Nor I. But -- as someone who's much less familiar with LLVM IR than you -- I assumed that if an idiot like me can guess this, it would be fairly obvious.

Reply:
Alright let's leave it then.